creating-joy/src/main/java/com/technocrats/creatingjoy/dto/UserDTO.java

```diff
@@ -1,4 +1,116 @@
 package com.technocrats.creatingjoy.dto;
 
+        import lombok.AllArgsConstructor;
+
+        import lombok.NoArgsConstructor;
+
+        import org.springframework.stereotype.Component;
+
+
+
+
+@NoArgsConstructor
+@AllArgsConstructor
 public class UserDTO {
-}
+
+
+    private String firstName;
+    private String lastName;
+    private String userName;
+    private String phoneNumber;
+    private String password;
+    private String houseNumber;
+    private String street;
+    private String city;
+    private String state;
+
+
+    public String getFirstName() {
+        return firstName;
+    }
+
+    public String getLastName() {
+        return lastName;
+    }
+
+    public String getUserName() {
+        return userName;
+    }
+
+    public String getPhoneNumber() {
+        return phoneNumber;
+    }
+
+    public String getPassword() {
+        return password;
+    }
+
+    public String getHouseNumber() {
+        return houseNumber;
+    }
+
+    public String getStreet() {
+        return street;
+    }
+
+    public String getCity() {
+        return city;
+    }
+
+    public String getState() {
+        return state;
+    }
+
+
+    public void setFirstName(String firstName) {
+        this.firstName = firstName;
+    }
+
+    public void setLastName(String lastName) {
+        this.lastName = lastName;
+    }
+
+    public void setUserName(String userName) {
+        this.userName = userName;
+    }
+
+    public void setPhoneNumber(String phoneNumber) {
+        this.phoneNumber = phoneNumber;
+    }
+
+    public void setPassword(String password) {
+        this.password = password;
+    }
+
+    public void setHouseNumber(String houseNumber) {
+        this.houseNumber = houseNumber;
+    }
+
+    public void setStreet(String street) {
+        this.street = street;
+    }
+
+    public void setCity(String city) {
+        this.city = city;
+    }
+
+    public void setState(String state) {
+        this.state = state;
+    }
+
+
+    @Override
+    public String toString() {
```

Comment:
Same.Use @ToString annotation and exclude the ones you don't want to include in toString.

Reply:
Done.